javascriptv3/example_code/sagemaker/scenarios/wkflw-sagemaker-geospatial-pipeline/messages.js

```diff
@@ -0,0 +1,29 @@
+export const MESSAGES = {
+  greeting: `Welcome to the Amazon SageMaker pipeline example scenario. This example workflow will guide you through setting up and executing a SageMaker pipeline. The pipeline uses an AWS Lambda function and an Amazon Simple Queue Service (Amazon SQS) queue. It also runs a vector enrichment reverse geocode job to reverse geocode addresses in an input file and store the results in an export file.`,
+  greetingHeader: "Welcome to Amazon SageMaker Pipelines.",
+  creatingRole: "Creating IAM role: ${ROLE_NAME}.",
+  roleCreated: "IAM role created: ${ROLE_NAME}.",
+  continue: "Continue?",
+  exit: "Exiting...",
+  attachPolicy: 'Attaching IAM policy "${POLICY_NAME}" to role "${ROLE_NAME}".',
+  policyAttached: "IAM Policy attached.",
+  creatingFunction: "Creating Lambda function: ${FUNCTION_NAME}.",
+  functionCreated: "Created Lambda function with name: ${FUNCTION_NAME}",
+  creatingSQSQueue: "Creating SQS queue: ${QUEUE_NAME}.",
+  sqsQueueCreated: "Created SQS queue with name: ${QUEUE_NAME}",
+  configuringLambdaSQSEventSource:
+    "Configuring ${LAMBDA_NAME} to receive events from ${QUEUE_NAME}",
```

Comment:
Some of these end in punctuation ".", some do not. Is that intentional?

Reply:
Added punctuation to all lines.